ansible/systemd/infinite.service

```diff
@@ -9,7 +9,7 @@ Group=infinite
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files up --detach"
-ExecStop=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files stop"
+ExecStop=/bin/bash -c "docker-compose --project-directory /home/infinite/docker-files down"
```

Comment:
So I ran this and unfortunately saw the same error, tried a couple of times, still kept seeing the same issue, however, after I went on the box and manually docker-compose down, to kill the only dangling container that was still running from before it started working and has succeeded several times since 🤦

Reply:
just to confirm:

```
  - name: enable systemd init for docker-compose
    become: true
    systemd:
      name: infinite
      enabled: true
      daemon_reload: true
```

This is what restarts the systemd service?

Does restart invoke both ExecStop and ExecStart and does it wait for the stop command to completely run before the start finishes?

Reply:
That action enabled installs the systemd unit file.

In the deployment playbook, other actions "notify" the action that restarts docker containers.